pkgs/by-name/yp/ypbind-mt/package.nix

```diff
@@ -0,0 +1,74 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  autoreconfHook,
+  docbook-xsl-nons,
+  libnsl,
+  libtirpc,
+  libxcrypt,
+  libxml2,
+  libxslt,
+  pkg-config,
+  rpcbind,
+  systemdLibs,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ypbind-mt";
+  version = "2.7.2";
+
+  nativeBuildInputs = [
+    autoreconfHook
+    docbook-xsl-nons
+    pkg-config
+    libxml2
+    libxslt
+  ];
+
+  buildInputs = [
+    libnsl
+    libtirpc
+    libxcrypt
+    rpcbind
+    systemdLibs
+  ];
+
+  src = fetchFromGitHub {
+    owner = "thkukuk";
+    repo = "ypbind-mt";
+    rev = "v${version}";
+    hash = "sha256-P8LH1vBTrNdgUQhzeixxBZR4atXyRVF+DuUbR6UeBJY=";
+  };
+
+  outputs = [ "out" ];
+
+  configurePhase = ''
```

Comment:
Can you explain your code here?

Reply:
Without the `configurePhase/installPhase` settings the derivation tried to install the files to the global root `/` directory.
 Has been fixed in my refactor commit [`97c8bec` (#333099)](https://github.com/NixOS/nixpkgs/pull/333099/commits/97c8bec445635a1a843d91f7d1eac67e274f9565)